### ISSUE TITLE
Use template gui without access to raw data

### ISSUE THREAD

**Issue by greggheller** [kwikteam/phy-contrib#142]
We were wondering if there is an easy way to open the template-gui without requiring access to the raw data. This would make it much easier and faster to move data back and forth between machines which is proving to be necessary for the high throughput experiments we are doing. This would also be useful when you want to compare the kilosort results for two different parameter sets. Obviously there are more rigorous ways to compare the sorting, but it would be nice to be able to pull them up side by side with two instances oh phy.

It seems like it shouldn't be too difficult to just "turn off" that portion of the waveform view so that you view only the templates, but I am not confident I could figure out the appropriate modifications. Any advice on how to do this?

If this seems like something more people could take advantage of, you could also include a heat map for each cluster's waveforms that helps preserve some of the variability information. Obviously this falls apart a bit if you are doing any splitting, but that would be a small price to pay in our use case.

**Comment by greggheller** [comment]
I have found a hacky way to do this - if you use a different data file phy will still load. If the dummy data file is shorter than the actual data file you get a bunch of "Error while loading waveform" (no surprise), but the gui will still load so everything works.

You still lose all mean waveform and waveform variability information, so heatmap/mean waveforms ouput by kilosort (or computed and added by the user) might still be a useful addition in the future, but this will definitely work for now.

**Comment by rossant** [comment]
What error message do you get if you just launch the GUI without the raw data? It is meant to work by disabling the parts of the GUI that use the raw data. If that doesn't work, it's a bug.

**Comment by greggheller** [comment]
Turns out it was the 'spike height' plugin I had running that was causing the problem. But without it it loads fine. Although I've definitely tried this before without any plugins, it must've been with an earlier/improperly installed version. 

I did notice that if you accidentally hit "m" when you are running things without any raw data the waveform view gets stuck. It will continue displaying whichever waveform was being viewed when you hit m. It throws an Attribute Error: NoneType object has no attribute mean every time you change clusters (all the other windows update just fine). It also throws this error if you try to change the waveform view in any way (toggling, zooming in or hitting w, or any other operation in the menu) so there is no way to get back to viewing templates except by reloading the gui.
